build/Dockerfile

```
@@ -61,6 +61,9 @@ RUN ln -s /etc/systemd/system/hostname-changer.service /etc/systemd/system/netwo
 # Set sddm as display manager and install Plasma as desktop environment
 RUN /bin/bash -c "debconf-set-selections <<< 'sddm shared/default-x-display-manager select sddm'" && apt-get -qq update && apt-get -qq install -y kde-plasma-desktop plasma-nm dolphin print-manager libglib2.0-dev locales kde-spectacle kcalc > /dev/null 2>&1
 
+# Install superimportant emojis
+RUN apt-get update -qq > /dev/null 2>&1 && apt-get install -y -qq fonts-noto-color-emoji
```

```suggestion
RUN apt-get update -qq > /dev/null 2>&1 && apt-get install -y -qq fonts-noto-color-emoji > /dev/null 2>&1
```
Any particular reason why this is integrated here and not in our customization repo?